Add stub for unittest.mock.patch.multiple()

srittau opened the pull request.

JelleZijlstra commented on stdlib/3/unittest/mock.pyi:
```diff
@@ -100,6 +100,7 @@ if sys.version_info >= (3, 3):
     class _patcher:
         def __call__(self, target: Any, new: Optional[Any] = None, spec: Optional[Any] = None, create: Any = False, spec_set: Optional[Any] = None, autospec: Optional[Any] = None, new_callable: Optional[Any] = None, **kwargs: Any) -> Any: ...
         def object(self, target: Any, attribute: str, new: Optional[Any] = None, spec: Optional[Any] = None, create: Any = False, spec_set: Optional[Any] = None, autospec: Optional[Any] = None, new_callable: Optional[Any] = None, **kwargs: Any) -> _patch: ...
+        def multiple(self, target: Any, spec: Optional[Any] = None, create: Any = False, spec_set: Optional[Any] = None, autospec: Optional[Any] = None, new_callable: Optional[Any] = None, **kwargs: Any) -> Any: ...
```
All default values should be `...` (even though the surrounding code doesn't follow that convention yet).

srittau commented:
I have updated the patch and added two more commits that change the arguments of `patch()` and `patch.object()` to the new style and tighten the type of the `create` argument, respectively.

JelleZijlstra commented:
Thanks, this looks good (relevant docs at https://docs.python.org/3/library/unittest.mock.html#patch-multiple). I'll merge once Travis is green.
